### PR TITLE
[Merged by Bors] - doc(FieldTheory/IsAlgClosed/Basic): add a TODO

### PR DESCRIPTION
**Pull request by acmepjz**
... which is: prove that if `K / k` is algebraic, and any monic irreducible polynomial over `k` has a root in `K`, then `K` is algebraically closed (in fact an algebraic closure of `k`).

Reference: <https://kconrad.math.uconn.edu/blurbs/galoistheory/algclosure.pdf>, Theorem 2. From the reference it looks like that the proof of this result needs purely inseparable argument, so probably it can't be done in this file.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

Discussions: https://github.com/leanprover-community/mathlib4/pull/9488#discussion_r1443865997

### CODE REVIEW

File: Mathlib/FieldTheory/IsAlgClosed/Basic.lean [hunk 1]
```diff
@@ -34,6 +34,13 @@ polynomial in `k` splits.
 
 algebraic closure, algebraically closed
 
+## TODO
+
+- Prove that if `K / k` is algebraic, any monic irreducible polynomial over `k` has a root in `K`,
```
Comment (jcommelin) [inline Mathlib/FieldTheory/IsAlgClosed/Basic.lean:39]:
```suggestion
- Prove that if `K / k` is algebraic, and any monic irreducible polynomial over `k` has a root in `K`,
```

**Review by jcommelin — COMMENTED** [review]
I suggest adding the word `and`, otherwise people (like me) might think you already started the "then" part of the claim.

bors d+

**Comment by acmepjz** [comment]
By the way, from the reference it looks like that the proof of this result needs purely inseparable argument, so probably it can't be done in this file.

**Pull request closed without merging**